data/human/campaign events.txt

```diff
@@ -519,6 +519,69 @@ event "initial deployment 3"
 		fleet "Large Republic" 800
 		fleet "Navy Surveillance" 1200
 
+
+
+mission "Central Outfitters Expansion Trigger"
+	landing
+	invisible
+	to offer
+		has "event: initial deployment 2"
+	on offer
+		event "central outfitters expansion" 45
+
+event "central outfitters expansion"
+	planet "Earth"
+		add outfitter "Central Outfits Expansion"
+	planet "Luna"
+		add outfitter "Central Outfits Expansion"
+	planet "New China"
+		add outfitter "Central Outfits Expansion"
+	planet "New Wales"
+		add outfitter "Central Outfits Expansion"
+	planet "Heartland"
+		add outfitter "Central Outfits Expansion"
+
+mission "Central Outfitters Expansion"
+	outfitter
+	source
+		planet "Earth" "Heartland" "Luna" "New China" "New Wales"
+	to offer
+		has "event: central outfitters expansion"
+		or
+			not "event: joined the free worlds"
+			not "event: start of hostilities"
```

Comment:
These events are applied at the same time.
What's the goal of this condition set?

Reply:
To ensure the player wouldn't hear contradictory info. But probably should just remove the or and the joined the free worlds.
```suggestion
		not "event: start of hostilities"
```